feat: Add missing `of` methods to scores in Python

Christopher-Chianelli opened the pull request.

Christopher-Chianelli committed:
feat: Add missing `of` methods to scores in Python

triceo reviewed (dismissed):
Please update the docs to use the new methods.
Otherwise LGTM.

Christopher-Chianelli committed:
docs: Update `of` method used in Python docs, fix Asciidoctor warning

Christopher-Chianelli committed:
docs: Use correct link

Asciidoctor refactor failed to update links, most likely due
to the duplicate id.
